test/arch/mep/jit/launch.py

```diff
@@ -0,0 +1,7 @@
+# Toshiba MeP-c4 - pytest unit tests wrapper
+# Guillaume Valadon <guillaume@valadon.net>
+
+from ut_helpers_jit import launch_tests
+
+from test_jit_branchjump import TestBranchJump; launch_tests(TestBranchJump())
+from test_jit_repeat.py import TestRepeat; launch_tests(TestRepeat())
```

Comment:
Tipo here. Remove the `.py` to be able to run the test

```python
from test_jit_repeat import TestRepeat; launch_tests(TestRepeat())
```

Reply:
Done!